config/manifests/bases/cluster-logging.clusterserviceversion.yaml

```diff
@@ -218,7 +379,6 @@ spec:
   maintainers:
   - email: aos-logging@redhat.com
     name: Red Hat
-  minKubeVersion: 1.18.3
```

Comment:
Is this not an indicator of the kube APIs we support?  Don't we need this to ensure we don't get deployed on a non-supporting cluster?

Reply:
I'll restore it for now. I think we should be checking API versions not minKubeVersions, I'll open a separate issue.